### ISSUE TITLE
[Feature] Need one function to test if dubbo-app is ready when doing rollout in kubernetes 

### ISSUE THREAD

**Issue by Hinsteny** [apache/dubbo#5872]
- [X] I have searched the [issues](https://github.com/apache/dubbo/issues) of this repository and believe that this is not a duplicate.
- [X] I have checked the [FAQ](https://github.com/apache/dubbo/blob/master/FAQ.md) of this repository and believe that this is not a duplicate.

### Environment

* Dubbo version: 2.7.7
* Operating System version: Mac OS
* Java version: jdk1.8
* kubernetes: 1.6

**Comment by chickenlj** [comment]
Yes, that's one feature we definitely need for Dubbo running on Kubernetes
